### PR TITLE
Fix the package name for eventratelimit config API

### PR DESCRIPTION
**Pull request by tengqm**

### CODE REVIEW

File: content/en/docs/reference/config-api/apiserver-eventratelimit.v1alpha1.md [hunk 1]
```diff
@@ -105,17 +105,17 @@ allowance of burst queries.</p>
...
-- [Limit](#evenratelimit-admission-k8s-io-v1alpha1-Limit)
+- [Limit](#eventratelimit-admission-k8s-io-v1alpha1-Limit)
 
 
 <p>LimitType is the type of the limit (e.g., per-namespace)</p>
 
 
 
-  
+
```
Comment (Sea-n) [inline content/en/docs/reference/config-api/apiserver-eventratelimit.v1alpha1.md:121]: Many files in `content/en/docs/reference/config-api` have this 3 newline + 2 space.

Not sure what should we do for this line.
1) Remove two spaces (current changes)
2) Leave it as is
3) Remove the entire line (newline + 2 space)

**Review by Sea-n — COMMENTED** [review]
Wow, it's an obvious typo, looks good to me!
/lgtm

File: content/en/docs/reference/config-api/apiserver-eventratelimit.v1alpha1.md (hunk 1 above)
Comment (tengqm) [reply content/en/docs/reference/config-api/apiserver-eventratelimit.v1alpha1.md:121]: This empty lines are generated from go-template.

**Review by sftim — COMMENTED** [review]
“Oops”

/approve

**Pull request merged**